src/stats.c

```diff
@@ -49,6 +49,12 @@ inline UInt EXEC_STAT(Stat stat)
     return (*STATE(CurrExecStatFuncs)[ tnum ]) ( stat );
 }
 
+inline Obj EXEC_STAT_RETURN_OBJ(Stat stat)
+{
+    UInt status = EXEC_STAT(stat);
+    GAP_ASSERT((status == 1) || (status == 2));
```

Comment:
Not sure this is necessary, the functions in `funcs.c` did not do anything with the return value of `EXEC_STAT`, anyway.